docs/source/troubleshooting/basic_chatops_troubleshooting.rst

```diff
@@ -117,7 +117,7 @@ either throws an error, or gives an acknowledgement message without result, or n
      aliases throw errors, the format of your alias or the underlying action is most
      likely the problem. Debug according to the error.
 
-5.  Bonus: have you tried turning it off and on again?
+5.  Bonus: have you tried turning BWC off and on again?
```

Comment:
This should not be BWC. It could be `|st2|`, so it gets replaced by StackStorm or BWC, as appropriate